src/app/search/search-filters.ts

```diff
@@ -494,6 +498,9 @@ export function searchFilters(
         } else if (term.startsWith('light:') || term.startsWith('power:')) {
           const filter = term.replace('light:', '').replace('power:', '');
           addPredicate('light', filter);
+        } else if (term.startsWith('masterwork:')) {
```

Comment:
I suppose my only question is - in the game, aren't these called "weapon tiers"?

Reply:
Hmm, I thought about adding the word tier to it, but then it felt too long, though we do have auto complete so it isn't that bad. There are also tiers for armor.

Reply:
I could alias weaponTier, armorTier, and masterwork, masterworkTier?

Reply:
They are called tier in-game, but only related to the masterwork status of an item.  There is no non-masterwork use of the term tier, so adding it just makes the filter name longer, but doesn't make it more useful.

We need to be consistent about how masterwork: and is:masterwork relate.  I made some comments about this in #3051

Reply:
ooh ok! so should i swap is:masterwork to instead be an alias for masterwork:=10? @philkernick @bhollis ?

Reply:
It’s unfortunately not that simple. For armor it’s masterwork:=5.  For exotics I don’t think the API returns a value at all, just a binary flag.

Reply:
Alright, I've just committed an update that should cover that.